tests/integrations/aws_lambda/test_aws.py

```diff
@@ -672,9 +672,10 @@ def test_serverless_no_code_instrumentation(run_lambda_function):
             import sentry_sdk
 
             def test_handler(event, context):
-                current_client = sentry_sdk.Hub.current.client
+                current_client = sentry_sdk.get_client()
 
                 assert current_client is not None
```

Comment:
```suggestion
```

`get_client()` always returns a non-`None` client object with the new API, so we can delete this check. The equivalent check in the new API is to simply check that the client is active, like you did on the next line.